pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -896,6 +897,7 @@ func (p *MasqueradePodInterface) createNatRulesUsingNftables(proto iptables.Prot
 			strings.ToLower(port.Protocol),
 			"dport",
 			strconv.Itoa(int(port.Port)),
+			Handler.GetNFTIPString(proto), "saddr", getLoopbackAdrress(proto),
```

Comment:
I did not try nftables, but would it also support the keyword `localhost`?

Reply:
same